src/core/agent/services/connectionService.ts

```diff
@@ -281,6 +278,7 @@ class ConnectionService extends AgentService {
       [`${KeriaContactKeyPrefix.CONNECTION_NOTE}${id}`]: JSON.stringify({
         ...note,
         id: `${KeriaContactKeyPrefix.CONNECTION_NOTE}${id}`,
+        timestamp: new Date(),
```

Comment:
Can we explicitly call `toISOString` here? just in case JSON.stringify impl changes

Reply:
I updated and tested all tracking flow before pushing